Source/WebCore/css/CSSFontFace.cpp

```diff
@@ -95,8 +95,16 @@ Ref<CSSFontFace> CSSFontFace::create(CSSFontSelector& fontSelector, StyleRuleFon
     return result;
 }
 
+static RefPtr<MutableStyleProperties> initialStyleProperties(StyleRuleFontFace* cssConnection)
+{
+    if (cssConnection)
+        return nullptr;
+    return MutableStyleProperties::create();
+}
+
 CSSFontFace::CSSFontFace(const Settings::Values* settings, StyleRuleFontFace* cssConnection, FontFace* wrapper, bool isLocalFallback)
-    : m_cssConnection(cssConnection)
+    : m_propertiesUnlessConnected(initialStyleProperties(cssConnection))
+    , m_cssConnection(cssConnection)
```

Comment:
The name `m_propertiesUnlessConnected` is a bit confusing, for someone reading this class at first sight, they might think a connection can be added to a `CSSFontFace` after it's been initialized, which is not the case. 

I'm not sure what would be a good name though, maybe `m_propertiesIfNotConnected` ?

Reply:
I’d be happy to change the name.

The existing `m_cssConnection` actually did make me think a connection could be added to `CSSFontFace` after it’s been initialized, so that concern is not new. I could use `std::variant` to combine `m_propertiesUnlessConnected` and `m_cssConnection` and make that data member `const`, but then what would I name it.

Reply:
I think it would help if `initialStyleProperties` was inlined: `m_propertiesIfNotConnected(cssConnection ? nullptr : MutableStyleProperties::create())` 

The combination of `initial` and `unless` make it confusing I think.

Reply:
I have a new version that uses `std::variant`. Came out pretty good.